libraries/SPI/SPI.cpp

```diff
@@ -511,6 +530,14 @@ void SPIClass::transferBytes(const uint8_t * out, uint8_t * in, uint32_t size) {
     }
 }
 
+/**
+ * Note:
+ *  in and out need to be aligned to 32Bit
+ *  or you get an Fatal exception (9)
+ * @param out uint8_t *
+ * @param in  uint8_t *
+ * @param size uint8_t (max 64)
+ */
 void SPIClass::transferBytes_(const uint8_t * out, uint8_t * in, uint8_t size) {
     while(SPI1CMD & SPIBUSY) {}
```

Comment:
I suggest an ASSERT() here checking the alignment of the inputs would be handy here and print a useful error when debugging was enabled...